gammapy/irf/energy_dispersion.py

```diff
@@ -924,3 +910,17 @@ def peek(self, figsize=(15, 5)):
         edisp.plot_matrix(ax=axes[2])
 
         plt.tight_layout()
+
+    def to_table(self):
+        #TODO : ETRUE or ENERG? See https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/edisp/index.html
```

Comment:
Please follow the spec (see [here](https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/edisp/index.html), i.e. use `ENERG_LO` and `ENERG_HI`.

We can discuss and make changes to the formats of course, or first introduce / prototype new formats in Gammapy / ctapipe and then write a spec for them after. But let's add support for the spec we have and have been using for current IACTs for a few years first.

Reply:
Will do (same as for the background), I am not sure why CTA has gone for slightly different definitions in their IRFs?

Reply:
> I am not sure why CTA has gone for slightly different definitions in their IRFs?

My 2 cents: CTA hasn't committed to any formats. What we have is preliminary and variants are due to missing communication in the past.  Concerning the specific issue, the discussion for CTA should continue here: https://forge.in2p3.fr/boards/236/topics/1824?r=2314#message-2314 gamma-astro-data-formats.readthedocs.io is just an attempt to write up the formats that we use, to have a format description at all and to avoid duplicating it (in this case in the ctapipe, Gammapy and Gammalib docs).

There is a lot of discussion and complaining,  but I think basically we just need more communication and collaboration between pipeline and science tool developers and things can be OK from here on out for CTA.

Reply:
I see, you were commenting on reco vs true energy axis names.
This has been discussed previously, e.g. here: https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/issues/63
I don't remember what I or others said, in principle I don't care much about FITS serialisation details. The problem with changing something is that it needs an active communication channel and collaboration, and as Karl suggested versioning. I hope that this will come soon, but for now just sticking with what we have in the spec (and used e.g. in HESS for years) seems simplest.